bin/release

```diff
@@ -3,6 +3,4 @@
 bundle exec rake build
 
 printf "\n✅ Lookbook v${1/-/.} gem built."
```

Comment:
We can remove these lines here as we no longer need to build the gem locally at all

```suggestion
```